packages/react-components/src/templates/__tests__/ToolModal.test.tsx

```diff
@@ -39,6 +39,50 @@ it('renders default values into inputs', () => {
   `);
 });
 
+it('allows url with https protocol', () => {
+  const { getAllByRole, queryByText } = render(<ToolModal {...props} />, {
+    wrapper: StaticRouter,
+  });
+  const inputUrl = getAllByRole('textbox')[0];
```

Comment:
Can we get by label text? [0] is quite brittle if we have to add another textbox